.github/workflows/rill-cloud.yml

```diff
@@ -40,17 +40,16 @@ jobs:
 
       - name: Build & Publish Rill docker image
         run: |-
+          echo "Fetch tags to get the last tagged version"
+          git fetch --all --tags;
+
           if [ "$RELEASE" == "true" ] || [ "$GITHUB_REF_NAME" == "main" ]; then
             echo "DEPLOY_CLOUD=1" >> $GITHUB_ENV
           else
+            echo "LATEST_BRANCH=$(git branch -r | grep release-0 | sort | tail -1)"
```

Comment:
I might just be missing something, but I don't see any code that references/uses `LATEST_BRANCH`?

Reply:
its a debug log